src/App.tsx

```diff
@@ -125,15 +125,13 @@ export const App: React.FC = () => {
           onError={setErrorMessage}
         />
         <TodoList list={todos} onDelete={onDeleteTodo} idTodo={idTodo} />
-        {todos.length > 0 && (
-          <Footer
-            onClick={setStatus}
-            status={status}
-            leftItems={leftItemsCount}
-            completedItems={completedItems}
-            onDelete={clearCompletedTodo}
-          />
-        )}
+        <Footer
```

Comment:
You should add conditional rendering here
But you should hide footer if there is no todos on server at all

Not when there are no todos with selected status:
![image](https://github.com/user-attachments/assets/a9c156b2-8ea1-456f-9e04-a33effda5dbc)